docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -820,9 +820,33 @@ Example response:
         "last_failure_timestamp" : null,
         "failures" : 0
       },
-      "queue" : {
-        "type" : "memory"
-      }
+      "queue": {
+        "type" : "persisted",
+        "capacity": {
+          "max_unread_events": 0,
+          "page_capacity_in_bytes": 67108864,
+          "max_queue_size_in_bytes": 1073741824,
+          "queue_size_in_bytes": 3885
+        },
+        "flow": {
```

Comment:
We have had an offline conversation about grouping all of our pipeline flow metrics under `pipeline.*.flow` with "leading context" naming that refines the context as you read the name from left to right. This would make it more straight-forward to get all of a pipeline's pipeline-level metrics in one place, without making the user dig in multiple places for a holistic pipeline-level view.

For example, our existing `input_throughput` first clues you in that we are talking about the `input` phase, and then defines that we are presenting its `throughput`.

Similarly `queue_persisted_growth_bytes` indicates first that it is about a pipeline’s `queue`, specifically one that is `persisted`, and its `growth` in `bytes`.

This means  that when you are scanning for meaningful information, you can abort reading the full metric name as quickly as possible if it is not meaningful for what you are looking for.

Reply:
+1! Another supportive thought, with `flow` metrics, we are providing metrics about **_what is going on_** with pipelines rather showing static **_what has_** metrics. So, placing them in a same place with flow would give better visibility on the condition.